src/Config/FileSystemRuntimeConfigLoader.cs

```diff
@@ -87,6 +89,32 @@ public string GetConfigFileName()
         return configFileName;
     }
 
+    /// <summary>
+    /// Checks if we have already attempted to configure the file watcher, if not
```

Comment:
Does it actually check if the config file watcher was instantiated previously? Not following from the code.